src/inverseRobot.js

```diff
@@ -25,6 +25,24 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const newRobot = {};
+
+  const keys = Object.values(robot);
+
+  const newKeys = [...new Set(keys)];
+
+  if (newKeys.length !== keys.length) {
+    return null;
+  }
+
+  Object.entries(robot).forEach(entry => {
+    const key = entry[1];
+    const value = entry[0];
+
+    newRobot[key] = value;
+  });
+
+  return newRobot;
```

Comment:
use `for in` loop and iterate through object
reverse  key and value by
`newObject[OldObject[key]] = key `